src/main/java/org/jboss/tools/intellij/openshift/utils/odo/OdoCli.java

```diff
@@ -130,7 +131,12 @@ Map of process launched for feature (dev, debug,...) related. Key is component n
         this.client = new DefaultKubernetesClient(new ConfigBuilder().build());
         try {
             this.envVars = NetworkUtils.buildEnvironmentVariables(this.getMasterUrl().toString());
-            this.envVars.put("ODO_DISABLE_TELEMETRY", "true");
+            if (TelemetryConfiguration.getInstance().isEnabled()) {
+                this.envVars.put("ODO_TRACKING_CONSENT", "yes");
+                this.envVars.put("TELEMETRY_CALLER", "intellij");
+            } else {
+                this.envVars.put("ODO_TRACKING_CONSENT", "no");
+            }
```

Comment:
What if the user chooses to enable/disable telemetry later (after `OcoCli` was instantiated)?
Isn't `"ODO_TRACKING_CONSENT"` duplicating `TelemetryConfiguration.getInstance().isEnabled()`?

Reply:
OdoCli is created each time a cluster is loaded so it will be reloaded next connection.
Is there a way to get notified of changes ?

Reply:
Yes there's a topic available. On should be able to subscribe to the topic and receive change notifications:
https://github.com/redhat-developer/intellij-redhat-telemetry/blob/690f872c66e63e64cfad9ec56d39bce33a0a7d36/src/main/java/com/redhat/devtools/intellij/telemetry/core/configuration/TelemetryConfiguration.java#L156-L161
```
    public interface ConfigurationChangedListener {
        Topic<ConfigurationChangedListener> CONFIGURATION_CHANGED =
                Topic.create("Telemetry Configuration Changed", ConfigurationChangedListener.class);


        void configurationChanged(String property, String value);
    }
```
I'm looking for usage examples.